website/static/js/osfHelpers.js

```diff
@@ -289,6 +289,19 @@ var urlParams = function(str) {
         .map(function(n){return n = n.split('='),this[n[0]] = decodeURIComponent(n[1]).replace(/\+/g, ' '),this;}.bind({}))[0];
 };
 
+/**
+  * Get query string arguments as an object.
+  * If `str` is falsy, return {}.
+  * Modified from getQueryParameters plugin by Nicholas Ortenzio (MIT Licensed).
+  */
+var urlEmail = function(str) {
```

Comment:
This probably isn't necessary, and it duplicates a lot from `urlParams`. See https://github.com/CenterForOpenScience/osf.io/pull/5528/files#r61776758